### PR TITLE
feat: harmony integarion

### PR DESCRIPTION
**Pull request by thelostone-mc**
##### Description

- integrates harmony into bounties
- added logo behind staff
- does minimal refactoring to code
- improves check to see if txn has already been recorded

**Pieces/Steps**
- [x] create bounty
- [x] sync fulfillment with chain
- [x] inject provider and connect to network
- [ ] actual transfer // this doesn't work due to bug in the extension - signing with private key to test for now
- [x] record activity

##### Testing

image

### CODE REVIEW

**Review by danlipert — APPROVED** [review]

File: app/assets/v2/js/pages/bounty_details2.js [hunk 1]
```diff
@@ -300,7 +308,7 @@ Vue.mixin({
         case 'ETC':
           tenant = 'ETC';
           break;
-        
+
```
Comment (chibie) [inline app/assets/v2/js/pages/bounty_details2.js:311]: @thelostone-mc

isn't this required?

```
case 'ONE':
  tenant = 'HARMONY';
  break;
```

Comment (thelostone-mc) [reply app/assets/v2/js/pages/bounty_details2.js:311]: derp not sure how i missed that ! fixed !
thanks 🙌

**Review by chibie — APPROVED** [review]

**Comment by danlipert** [comment]
@thelostone-mc mark as draft until they get back to us?

**Comment by thelostone-mc** [comment]
@danlipert i'd lean towards getting this in cause and have it ready behind a staff flag

**Pull request merged**